front/lib/api/assistant/citations.ts

```diff
@@ -221,10 +221,7 @@ export const getLightAgentMessageFromAgentMessage = (
     chainOfThought: agentMessage.chainOfThought,
     error: agentMessage.error,
     status: agentMessage.status,
-    actions: agentMessage.actions.map((a) => ({
-      type: a.type,
-      id: a.id,
-    })),
+    actions: agentMessage.actions,
```

Comment:
The main purpose of LightAgentMessage is to remove all actions from the payload output. Putting all actions details back in the message payload will just move us back to the previous state. ( https://app.datadoghq.eu/notebook/220988/fetch-message-latency?range=604800000&start=1749455514447&live=true )